runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -188,29 +186,33 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
             }
         }
 
-        // Linearize this read against a timestamp
-        final long timestamp =
-                rt.getSequencerView()
-                        .nextToken(Collections.singleton(streamID), 0).getToken()
-                        .getTokenValue();
-        log.debug("Access[{}] conflictObj={} version={}", this, conflictObject, timestamp);
+        // try to perform the underlying access. Trim failures can happen due to fast
```

Comment:
Can you leave a note that this is temporary until checkpointing is completed?

Reply:
done.